### DISCUSSION TITLE
Adding a song to a collection from results screen

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#28597] [Ideas] [answered]
Hope it's not a duplicate, couldn't find a matching discussion.
Basically, it would be nice to have a button on the bottom bar in result screen to directly add a song to a collection.
Feels like a gain of time, and would avoid switching to mouse in order to use the right click menu in song select.
Thanks for considering my proposal :)

**Comment by peppy** [comment]
I'd definitely like to see this happen, alongside the ability to favourite a beatmap.

There's room for new buttons in the footer at the bottom, if anyone wants to give it a go.

**Comment by Layendan** [comment]
Assigning this to myself, already added the favorite button, will add the collection button as well
![osu_2024-07-17_12-57-06](https://github.com/user-attachments/assets/ea41a5f5-a562-4d0d-816a-072741d2aa30)

**Reply by Layendan** [reply, marked as answer]
https://github.com/ppy/osu/pull/28991
